srcpkgs/booster/template

```diff
@@ -0,0 +1,38 @@
+# Template file for 'booster'
+pkgname=booster
+version=0.3
+revision=1
+build_style=go
+go_import_path=github.com/anatol/booster
+go_package="${go_import_path}/init ${go_import_path}/generator"
+go_mod_mode=readonly
+hostmakedepends="git"
+depends="busybox-static"
+short_desc="Fast and secure initramfs generator"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://github.com/anatol/booster"
+distfiles="https://github.com/anatol/booster/archive/${version}.tar.gz"
+checksum=77067a5056e1070aee0f27130c31ebbf7cacca18644dc4906290371a04f07371
+
+do_build() {
+	cd generator
+	CGO_CPPFLAGS="${CPPFLAGS}" CGO_CFLAGS="${CFLAGS}" CGO_CXXFLAGS="${CXXFLAGS}" CGO_LDFLAGS="${LDFLAGS}" \
+	go build -trimpath -buildmode=pie -mod=readonly -modcacherw -ldflags "-linkmode external -extldflags"
```

Comment:
Any idea on how to cross-compile this? I'm not really familiar with why these options are needed to build this.